<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="inputTextEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>581</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RSA + ECC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateRsaButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tạo chữ ký RSA</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="rsaSignatureEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>581</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="verifyRsaButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Xác minh RSA</string>
    </property>
   </widget>
   <widget class="QLabel" name="rsaStatusLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="generateEccButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tạo chữ ký ECC</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="eccSignatureEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>631</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="verifyEccButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>620</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Xác minh ECC</string>
    </property>
   </widget>
   <widget class="QLabel" name="eccStatusLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>680</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
